mtp_noms_ops/assets-src/stylesheets/views/_tags.scss

```diff
@@ -0,0 +1,9 @@
+.rejected-tag {
```

Comment:
could you prefix classes with `mtp-` so that we know they're our styles and not from a third party pls